assets/quiz.json

```
@@ -0,0 +1,102 @@
+[
+  {
+    "number": 1,
+    "question": "Um wieviel Uhr ist die Küchenuhr stehen geblieben?",
+    "answers": [
+      { "answer": "Antwort 1", "isTrue": true, "key": 1 },
+      { "answer": "Antwort 2", "isTrue": false, "key": 2 },
+      { "answer": "Antwort 3", "isTrue": false, "key": 3 },
+      { "answer": "Antwort 4", "isTrue": false, "key": 4 }
+    ]
```

Putting all my comments together you could do something like this:

```suggestion
    "number": 1,
    "question": "Um wieviel Uhr ist die Küchenuhr stehen geblieben?",
    "correctAnswerIndex": 0,
    "answers": [ "Antwort 1", "Antwort 2", "Antwort 3", "Antwort 4" ]
    ]
```